metricbeat/module/golang/heap/_meta/data.json

```diff
@@ -0,0 +1,47 @@
+{
+    "@timestamp":"2016-05-23T08:05:34.853Z",
+    "beat":{
+        "hostname":"beathost",
+        "name":"beathost"
+    },
+    "metricset":{
+        "host":"localhost",
+        "module":"golang",
+        "name":"heap",
+        "rtt":44269
+    },
+    "golang":{
+        "heap":{
+            "cmdline": [
+                "./gopa"
+            ] ,
+            "alloc": 1759408,
+            "buck_hash_sys": 1466577,
+            "frees": 718931,
+            "gc_cpu_fraction": 0.000002,
+            "gc_sys": 440320,
+            "heap_alloc": 1759408,
```

Comment:
Make sure to follow the naming conventions: https://www.elastic.co/guide/en/beats/libbeat/5.2/event-conventions.html